src/bench/verify_script.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include <bench/bench.h>
 #include <key.h>
-#if defined(HAVE_CONSENSUS_LIB)
+#if HAVE_CONSENSUS_LIB
```

Comment:
> Conditional calls to AC_DEFINE (such as those setting it to a constant `1`, or not setting a value at all) need `defined()`/`#ifdef` AIUI

Agree with @luke-jr.

In this particular case, it is conditional: https://github.com/bitcoin/bitcoin/blob/8126551d54ffd290ed5767248be4b3d19243787b/configure.ac#L1707-L1710

therefore, the current code is fine.